tests/foreman/api/test_convert2rhel.py

```diff
@@ -261,8 +261,18 @@ def test_convert2rhel_oracle_with_pre_conversion_template_check(
     """
     major = version.split('.')[0]
     assert oracle.execute('yum -y update').status == 0
+    if oracle.execute('needs-restarting -r').status == 1:
+        oracle.power_control(state='reboot')
 
     if major == '8':
+        # Set RHEL kernel to be used during boot
+        oracle.execute("mkdir -p /boot/loader/entries/backup")
+        oracle.execute("mv /boot/loader/entries/*uek*.conf /boot/loader/entries/backup/")
+        # Needs reboot to reflect the changes
+        oracle.power_control(state='reboot')
+        assert oracle.execute("grubby --default-kernel | grep uek").status != 0
+        assert oracle.execute("uname -r | grep uek").status != 0
```

Comment:
Since we're configuring to use non-UEK or RHEL kernel in the setup fixture itself, so can we just move this workaround and this assertions into the fixture?

Reply:
So in the setup when we do yum update, can we move it after that as we've reboot step there too?

Reply:
We can but we need to run this step just or OEL 8 so need to add that condition there. Since we already had that condition in test so I added these steps in test itself.

Reply:
yes, and same can be moved to setup fixture where we add a check for OEL8, as its more of a setup part and not related to conversion test

Reply:
Yes, that I can do. Updated.